src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -227,7 +227,8 @@ class MultiDiracDeterminant : public WaveFunctionComponent
   void createDetData(const ci_configuration2& ref,
                      std::vector<int>& data,
                      std::vector<std::pair<int, int>>& pairs,
-                     std::vector<RealType>& sign);
+                     std::vector<RealType>& sign,
+                     std::vector<size_t>& C2nodes_ptcl);
```

Comment:
Please document this function with all its arguments.

Reply:
I added some documentation (I changed as little as possible from the previous implementation, but I don't know why we pass around so many objects that are already members of the `MultiDiracDeterminant`)

Reply:
Current APIs and data encapsulations are messy and I would like to tide them up later.
Sometimes we do pass class member via argument, in such way, the function may be turned const or event static to make the function saner and independently testable.